src/lib/api/vaults/convertToApy.ts

```diff
@@ -1,7 +1,19 @@
-export function convertToApy(apr: number | undefined) {
+export function convertToApy(apr?: number) {
   const compoundPeriod = 84_600 * 7 // 7 days - 1 week
   const yearInSecond = 31_556_926
   const n = yearInSecond / compoundPeriod
 
   return apr === undefined ? 0 : (1 + apr / n) ** n - 1
 }
+
+export function concentratorConvertToApy(
+  concentratorApr?: number,
+  compounderApy?: number
+) {
+  const compoundPeriod = 84_600 * 7 // 7 days - 1 week
+  const yearInSecond = 31_556_926
+  const n = yearInSecond / compoundPeriod
+  return concentratorApr === undefined || compounderApy === undefined
+    ? 0
+    : concentratorApr * (((1 + compounderApy / n) ** n - 1) / compounderApy)
```

Comment:
Should you be converting `compounderApy` to APY? Isn't it already APY?

If all you're doing is multiplying two APY values I don't think this needs a dedicated function. Just do this math where it's needed. 

If you really want to keep it as a dedicated function you should at least reuse `convertApy` here instead of duplicating the logic.

Reply:
The compounderApy is already an APY. I reuse the convertApy